docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -299,6 +300,14 @@ testcafe chrome my-tests --app 'node server.js'
 
 Use the [--app-init-delay](#--app-init-delay-ms) option to specify the amount of time allowed for this command to initialize the tested application.
 
+### -d, --debug-mode
+
+Specify this option to run tests in the debugging mode. In this mode, test execution is paused before the first action or assertion allowing you to invoke the developer tools and debug.
+
+In the footer, a status bar is displayed in which you can resume test execution or step to the next action or assertion.
```

Comment:
There is `The footer displays buttons that allow you to continue test execution or step to the next test action.` in our [`Debugging` section](https://devexpress.github.io/testcafe/documentation/test-api/debugging.html#client-side-debugging). No word about assertions. May be we need to add this?